package.json

```diff
@@ -54,6 +54,7 @@
     "tap": "^10.2.0",
     "tiny-worker": "^2.1.1",
     "webpack": "^2.4.1",
-    "webpack-dev-server": "^2.4.1"
+    "webpack-dev-server": "^2.4.1",
+    "worker-loader": "0.8.1"
```

Comment:
This should probably use a fuzzy (`^`) version number.

Reply:
I thought we were pinning exact versions for packages which have direct impact on the build output, which `worker-loader` does. Has that advice changed?